### ISSUE TITLE
Bug: FlexPaper (Standard) / HTML4 Mode: Window height shrinks when returning from fullscreen mode

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fleszar1/flexpaper#314]
```
FlexPaper (Standard) / HTML4 Mode: Window height shrinks when returning from 
fullscreen mode.

How to test:

-- Open a doc in HTML4 mode, e.g. this 
http://devaldi.com/adaptiveui/Intl_Monetary_Fund.php?ro=html,flash&prefer=true

-- Click on fullscreen

-- Scroll a bit down

-- Press escape key


-> Height of FlexPaper Window (id="pagesContainer_documentViewer" 
class="flexpaper_pages") shrinked in case of the above mentioned document from 
"height: 463px" to "height: 446px"
```

Original issue reported on code.google.com by `mentur...@gmail.com` on 13 Oct 2012 at 4:59

**Comment by GoogleCodeExporter** [comment]
```
Addition: 
When repeating the above test scenario subsequently the window height shrinks 
again and again.
```

Original comment by `mentur...@gmail.com` on 13 Oct 2012 at 5:06

**Comment by GoogleCodeExporter** [comment]
Original comment by `erik.eng...@devaldi.com` on 15 Oct 2012 at 9:42
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
fixed in upcoming build 2.02
```

Original comment by `erik.eng...@devaldi.com` on 17 Oct 2012 at 6:26
- Changed state: **Fixed**
